README-CONAN.md

```diff
@@ -117,25 +202,208 @@ os_build=Windows
+|:---------|--------------------|--------------------|
+| 64 bit | "Visual Studio 14 2015 Win64"    | "Visual Studio 15 2017 Win64" |
+| 32 bit | "Visual Studio 14 2015"    | "Visual Studio 15 2017" |
+
+### Recommended settings for Visual Studio
+
+
+##### 64 bit Release Build
+
+| | Visual Studio 2015 | Visual Studio 2017|
+|:---------|--------------------|--------------------|
+| _**conan install .. --profile**_ | msvc2015Release64 | msvc2017Release64 |
+| _**cmake -G**_                   |  "Visual Studio 14 2015 Win64"    | "Visual Studio 15 2017 Win64" |
+| _**cmake**_                      | -DCMAKE\_BUILD\_TYPE=Release | -DCMAKE\_BUILD\_TYPE=Release |
+| _**profile**_ | arch=x86\_64  | arch=x86\_64 |
+| | arch\_build=x86\_64 | arch\_build=x86\_64 |
+| | build\_type=Release | build\_type=Release |
+| | compiler.runtime=MD | compiler.runtime=MD |
+| | compiler.version=14 | compiler.version=15 |
+| | compiler=Visual Studio | compiler=Visual Studio |
+| | os=Windows | os=Windows |
+| | os\_build=Windows | os\_build=Windows |
+
+##### Debug Builds
+
+|| Visual Studio 2015 | Visual Studio 2017|
+|:-------|-------|------|
+| _**conan install .. --profile**_ | msvc2015Debug64 | msvc2017Debug64 |
+| _**cmake**_ | -DCMAKE\_BUILD\_TYPE=Debug | -DCMAKE\_BUILD\_TYPE=Debug |
+| _**profile**_ | build_type=Debug | build_type=Debug |
+| | compiler.runtime=MDd | compiler.runtime=MDd |
+
+##### 32bit Builds
+
+|| Visual Studio 2015 | Visual Studio 2017|
+|:-----------|--------------------|--------------------|
+| _**conan install .. --profile**_ | msvc2015Release32 | msvc2017Release32 |
+| _**cmake -G**_ | "Visual Studio 14 2015" | "Visual Studio 15 2017" |
+| _**profile**_ | arch=x86 | arch=x86 |
+| | arch_build=x86 | arch_build=x86 |
+
+### Modifying build settings in the Visual Studio IDE
+
+**CAUTION** You can change the runtime library in Visual Studio once you open the
+solution generated by CMake.  If you have generated a project to use the dynamic run-time library
+with the profile setting `compiler.runtime=MD` (or MDd), Visual Studio allows you to change that setting.
+However the libraries (provided by conan) to which you link are tied
+to a specific configuration and you will obtain linking errors.
+
+If you want to work with the static runtime library `MT` (or `MTd`), set up the profiles and build in a different directory such as build-static.
+
+### Changing profile settings with the conan command
+
+It is recommended that you create 8 profiles for msvc{2017|2015}{Release|Debug}{64|32}.
+
+You can modify profile settings on the command line.
+The following example demonstrates making substantial changes to profile settings by performing a 32 bit build using Visual Studio 2015 with a 2017 profile!
+
+```bash
+$ conan install .. --profile msvc2017Release64 -s arch_build=x86 -s arch=x86 -s compiler.version=14
+$ cmake         .. -G "Visual Studio 2015"
+$ cmake --build .  --config Release
+```
+
+### CMake/Cygwin Collisions on the build machine
```

Comment:
As we have discussed in Slack, I think this is caused by your `%PATH%` configuration. I installed Cygwin in my windows machine, and I did not have these issues.

I do not think it is a good idea to include these notes in the CONAN readme. It has nothing to do with conan, and it is not a generic situation, but a particular case for an erroneous user system configuration. We should try to keep this documentation as short as possible, and focused on CONAN.

Reply:
I put in more time on this.

I have several c:\\cygwin64 directories on my %PATH%.  Remarkably the issue is not present when c:\\cygwin64\\bin is on the path.  I think it's important to document that there is an issue with c:\\cygwin64 and the path.  This is where the user will encounter the issue and it's good to explain it exactly where it will be experienced.

Reply:
It's fine if you want to document this case for cygwin users. However I think it would make more sense to write that information in the `README-CMAKE` rather than in this document. It's a problem in the CMake configuration step, and it has nothing to do with conan.